src/beforeBuild.ts

```diff
@@ -97,3 +97,30 @@ await mkdir(COMPILED_INSTALLATION_SPELL_AQUA_WITH_TRACING_PATH, {
 
 await compileInstallationSpellAqua();
 await compileInstallationSpellAqua(true);
+
+const BIN_FILE_PATH = join(
+  join("node_modules", "oclif", "lib", "tarballs"),
+  "bin.js",
+);
+
+const binFileContent = await readFile(BIN_FILE_PATH, FS_OPTIONS);
+const NODE_RUN = `  "\\$NODE" "\\$DIR/run" "\\$@"`;
```

Comment:
What if oclif will change this line in the next version?

Reply:
Most likely the error will be thrown, as you can see from the code further down below